Samples/Desktop/D3D12Raytracing/readme.md

```diff
@@ -1,16 +1,38 @@
-
 # D3D12 Raytracing Samples
-This collection of samples act as an introduction to Direct3D 12 Raytracing. Each sample introduces a few new concepts. 
+This collection of samples act as an introduction to DirectX Raytracing (DXR). The samples are divided into tutorials and advanced samples. Each tutorial sample introduces a few new concepts. Advanced samples demonstrate more complex techniques and applications of raytracing. 
+
+The samples implement both DXR and D3D12 Raytracing Fallback Layer APIs. This is purely for demonstration purposes to show where the API differences are. Real-world applications will implement only one or the other. The Fallback Layer uses DXR if a driver and OS supports it. Otherwise, it falls back to compute pipeline to emulate raytracing. Developers aiming for a wider HW support should target the Fallback Layer.
```

Comment:
nit: I'd condense "to show where API difference are" -> "to show API differences"

Reply:
"it falls back to compute pipeline" -> "it falls back to the compute pipeline"

Reply:
"for a wider HW support" -> "for wider HW support"